<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sistema de Gestión Farmacéutica</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSistema">
    <property name="title">
     <string>Sistema</string>
    </property>
    <addaction name="actionInventario"/>
    <addaction name="actionVentas"/>
    <addaction name="actionProveedores"/>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
   </widget>
   <addaction name="menuSistema"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionInventario">
   <property name="text">
    <string>Inventario</string>
   </property>
  </action>
  <action name="actionVentas">
   <property name="text">
    <string>Ventas</string>
   </property>
  </action>
  <action name="actionProveedores">
   <property name="text">
    <string>Proveedores</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
